fil-proofs-tooling/scripts/benchy-remote.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMDS=$(cat <<EOF
+cd \$(mktemp -d)
+git clone -q https://github.com/filecoin-project/rust-fil-proofs.git
+cd rust-fil-proofs
+git checkout -q master
+/root/.cargo/bin/cargo build --release --all > /dev/null 2>&1
```

Comment:
we should always build with `RUSTFLAGS="-C target-cpu=native"` for benchmarks

Reply:
Done!